src/arena/Sprint.js

```diff
@@ -280,61 +312,52 @@ function Sprint(props) {
                         }
                       >
                         Save
-                      </button>
+                      </Button>
                     </div>
                   </div>
                 )
               )}
             </>
           </TabPanel>
-          <TabPanel value={key} index={1} style={{ margin: -30 }}>
+          <TabPanel value={key} index={1}>
             <>
+              <h5>{getFormattedDay()}</h5>
               <div>
                 <h2>My Stats</h2>
 
                 <div
                   style={{
                     display: "flex",
                     justifyContent: "start",
-                    marginTop: 16,
+                    margin: "16px -8px",
                     flexWrap: "wrap",
                   }}
                   className="first-step-arena"
                 >
-                  <div className="block">
-                    <p>
-                      {
-                        props.redux.sprint[activeSprintId].teams[index]
-                          .missions[day].dailyScore
-                      }{" "}
-                      {I18n.get("dailyPoints")}
-                    </p>
-                  </div>
-                  <div className="block">
-                    <p>
-                      {props.redux.sprint[activeSprintId].teams[index].missions[
-                        day
-                      ].dailyCompletion.toFixed(2)}
-                      % {I18n.get("daily")} CP
-                    </p>
-                  </div>
-
-                  <div className="block">
-                    <p>
-                      {props.redux.sprint[activeSprintId].teams[index].score}{" "}
-                      {I18n.get("weeklyPoints")}{" "}
-                    </p>
-                  </div>
-
-                  <div className="block">
-                    <p>
-                      {
-                        props.redux.sprint[activeSprintId].teams[index]
-                          .percentage
-                      }
-                      % {I18n.get("weekly")} CP
-                    </p>
-                  </div>
+                  <StatsBlock
```

Comment:
Great work proactively creating a re-usable component - will make styling/theming/flexing much easier to update.